### PR TITLE
POD-2213: Add Terra Utils Integration Tests

### PR DESCRIPTION
**Pull request by sahakiann**
[POD-2213](https://broadinstitute.atlassian.net/browse/POD-2213): Add Terra Utils Integration Test

Summary of Changes: 
* Integration tests were added for the Terra utilities (only for GCP related methods, _not Azure_) 
* Some small changes were made to methods that were previously not returning anything (including responses or status codes) 

[POD-2213]: https://broadinstitute.atlassian.net/browse/POD-2213?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: python/tests/integration_tests/test_terra_util.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+from python.utils.request_util import RunRequest
+from python.utils.terra_utils.terra_util import TerraWorkspace, TerraGroups, MEMBER
+from python.utils.terra_utils.terra_workflow_configs import WorkflowConfigs
+from python.utils.token_util import Token
+
+INTEGRATION_TEST_TERRA_BILLING_PROJECT = "ops-integration-billing"
+INTEGRATION_TEST_TERRA_WORKSPACE_NAME = "ops-integration-test-workspace"
+INTEGRATION_TEST_GROUP_NAME = "ops-integration-test-group"
+
+auth_token = Token(cloud="gcp")
```
Comment (snovod) [inline python/tests/integration_tests/test_terra_util.py:15]: Could we import GCP from util init to keep consisitent?

**Review by snovod — APPROVED** [review]
Looks good. one small comment.

File: python/tests/integration_tests/test_terra_util.py (hunk 1 above)
Comment (sahakiann) [reply python/tests/integration_tests/test_terra_util.py]: Good catch, updated!

**Pull request merged**